clang/lib/CodeGen/BackendUtil.cpp

```diff
@@ -795,12 +795,29 @@ static void addSanitizers(const Triple &TargetTriple,
     PB.registerOptimizerLastEPCallback(SanitizersCallback);
   }
 
-  if (LowerAllowCheckPass::IsRequested()) {
+  bool lowerAllowCheck = LowerAllowCheckPass::IsRequested();
+  // Is there a non-zero cutoff?
+  static constexpr double SanitizerMaskCutoffsEps = 0.000000001f;
```

Comment:
threshold is already done in SanitizeSkipHotCutoffs::operator[]

Reply:
Removed